torchaudio/models/decoder/_ctc_decoder.py

```diff
@@ -113,12 +113,78 @@ class CTCHypothesis(NamedTuple):
     timesteps: torch.IntTensor
 
 
+class CTCDecoderLMState(_LMState):
+    """Language model state.
+
+    :ivar Dict[int] children: Map of indices to LM states
+    """
+
+    def child(self, usrIdx: int):
+        """Returns child corresponding to usrIdx, or creates and returns a new state if input index
+        is not found.
+
+        Args:
+            usrIdx (int): index corresponding to child state
+
+        Returns:
+            CTCDecoderLMState: child state corresponding to usrIdx
+        """
+        pass
```

Comment:
Is the base class (and this class) intended to be an interface? If so, let's decorate it with [`abc.abstractmethod`](https://docs.python.org/3/library/abc.html#abc.abstractmethod).